plumbing/transport/http/testdata/certs/server.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
What are your thoughts on just generating the cert files on demand before running the tests?
Main idea was to make it clear how it is being generated, so it is easier for folks to maintain it in the future. Happy to be shot down.

Reply:
i think that might increase the time to run tests considerably as generating these certs does take a non-negligible amount of time. maybe we could add a README or help.txt in the folder containing the commands that were used to generate them?

Reply:
Sure, or maybe add as part of the commit description.